src/components/ui/EditIcon/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { BLACK } from '../../../settings/__color';
+import { fontSizes } from '../../../settings/__fonts';
+
+interface EditIconProps {
+  children: React.ReactNode;
+}
+function EditIcon(props: EditIconProps): React.ReactElement<EditIconProps> {
+  return <EditIcon.Wrapper>{props.children}</EditIcon.Wrapper>;
+}
+
+EditIcon.Wrapper = styled.div`
+  i {
+    color: ${BLACK};
+    font-size: ${fontSizes.large};
+    &:hover {
+      cursor: pointer;
+    }
+  }
+`;
+export default EditIcon;
```

Comment:
We now have an `Icon` on the codebase. Look at the `ui/` folder.

Reply:
resolved

Reply:
Awesome.